### DISCUSSION TITLE
for help

### DISCUSSION THREAD

**Discussion by Xie-Nav** [community/community#32617] [Other Feature Feedback, Questions, & Ideas]

**Comment by airtower-luna** [comment]
You post sounds like you're trying to message a specific person, but this is a public forum, where it'd be pure luck if that person sees your message. I hope you can find a better way to reach them!

**Comment by kat125** [comment]
On Sat, Sep 10, 2022 at 10:22 PM xwaiyy123 ***@***.***> wrote:

> Dear Mr Chaplot
>
> I have read your paper "Chaplot, Devendra Singh, et al. "Neural
> topological slam for visual navigation." Proceedings of the IEEE/CVF
> Conference on Computer Vision and Pattern Recognition. 2020.",I found this
> work very interesting.
>
> can you share me the code for this paper?
>
> thanks!
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/community/discussions/32617>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/A2CBO24WQT32TOUKJQ7JTWDV5VF6DANCNFSM6AAAAAAQJS3L5M>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>
I don’t know how still learning
